multistep/commonsteps/step_download.go

```diff
@@ -56,12 +57,39 @@ type StepDownload struct {
 }
 
 var defaultGetterClient = getter.Client{
-	Getters: getter.Getters,
-}
-
-func init() {
-	defaultGetterClient.Getters = append(defaultGetterClient.Getters, new(gcs.Getter))
-	defaultGetterClient.Getters = append(defaultGetterClient.Getters, new(s3.Getter))
+	// Disable writing and reading through symlinks
+	DisableSymlinks: true,
+	// The order of the Getters in the list may affect the result
+	// depending if the Request.Src is detected as valid by multiple getters
+	Getters: []getter.Getter{
+		&getter.GitGetter{
+			Timeout: 5 * time.Minute,
+			Detectors: []getter.Detector{
+				new(getter.GitHubDetector),
+				new(getter.GitDetector),
+				new(getter.BitBucketDetector),
+				new(getter.GitLabDetector),
+			},
+		},
+		&getter.HgGetter{
+			Timeout: 5 * time.Minute,
+		},
+		new(getter.SmbClientGetter),
```

Comment:
🤔 Should this getter have a timeout? I see this getter only exists in v2, and doesn't have that option currently. Just curious to your thoughts.

Reply:
Yeah - I think it should. It was not caught in the go-getter updates. I will follow up with a separate PR for that change.

Reply:
PR opened for the SmbClientGetter https://github.com/hashicorp/go-getter/pull/369